openslides/motion/templates/motion/slide.html

```diff
@@ -77,6 +77,9 @@ <h4 class="first">{% trans "Status" %}:</h4>
     {{ motion.active_version.title }}
     <small>
         {% trans "Motion" %} {{ motion.identifier|default:'' }}
+        {% if motion.is_amendment %}
+            (Amendment of <strong>{{ motion.parent }}</strong>)
```

Comment:
Please remove the strong-tag in slides subtitle (we never use bold style in subtitle).

And please reduce it to motion.parent.identifier (as i proposed). Otherwise you get multiple line breaks in subtitle if you have a long main motion. The identifier is enough. The title is (normally) the same like in the amendment.